hello-world/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+
+
+
+    defaultConfig {
+        applicationId "com.example.hello_world"
```

Comment:
change package and application id to `kin.sdk.hello_world` for consistency

Reply:
It is not part of the sdk